### ISSUE TITLE
PendingDeprecationWarning: lib2to3 package is deprecated and may not be able to parse Python 3.10+

### ISSUE THREAD

**Issue by juliangilbey** [google/yapf#888]
I received the following warning when using Python 3.9:

```
  /usr/lib/python3/dist-packages/yapf/__init__.py:35: PendingDeprecationWarning: lib2to3 package is deprecated and may not be able to parse Python 3.10+
    from lib2to3.pgen2 import tokenize
```

I'm sure you're aware already, but just in case not....

**Comment by tirkarthi** [comment]
See also https://github.com/google/yapf/issues/448 for a discussion of alternate libraries.

**Comment by bwendling** [comment]
Closed with 7c408b9d7750292760ed255f744211d1ef535668.

**Comment by redwan-islam-rumman** [comment]
I need this project
